external/webserver/http_client.c

```diff
@@ -305,6 +365,9 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 						if (strcmp(key, "Sec-WebSocket-Key") == 0) {
 							strncpy((char *)client->ws_key, value, WEBSOCKET_CLIENT_KEY_LEN);
 						}
+						if (strcmp(key, "Keep-Alive") == 0) {
+							parse_keep_alive_header(client, params);
```

Comment:
It seems good approach to define new function to parse keep alive header for clean code